packages/mui-base/src/Popover/Positioner/usePopoverPositioner.tsx

```diff
@@ -9,13 +9,15 @@ import type {
 import { mergeReactProps } from '../../utils/mergeReactProps';
 import { useAnchorPositioning } from '../../utils/useAnchorPositioning';
 import type { GenericHTMLProps } from '../../utils/types';
-import { getInertValue } from '../../utils/getInertValue';
```

Comment:
Is `getInertValue` still used anywhere? If not, we can remove it.

Reply:
Looks like it's used for `CheckboxRoot` input, but given it has `tabIndex: -1` and `aria-hidden: true`, I don't think that's necessary?